### PR TITLE
add is_moderator_marked to comments

### PR DESCRIPTION
**Pull request by fuzzylogic2000**

### CODE REVIEW

File: adhocracy4/comments/models.py [hunk 1]
```diff
@@ -45,6 +45,10 @@ class Comment(base.UserGeneratedContentModel):
         null=True,
         editable=False
     )
+    is_recommended = models.BooleanField(
```
Comment (treba123) [inline adhocracy4/comments/models.py]: wrong wording ;)

Comment (fuzzylogic2000) [reply adhocracy4/comments/models.py]: But that's why WIP!

Comment (treba123) [reply adhocracy4/comments/models.py]: ah alright, I thought wenn you added me to reviewers you wanted me to actually look into it :)

File: adhocracy4/comments/api.py [hunk 2]
```diff
@@ -52,3 +53,29 @@ def destroy(self, request, *args, **kwargs):
...
+
+class CommentModerateSet(mixins.CreateModelMixin,
+                         mixins.RetrieveModelMixin,
+                         mixins.UpdateModelMixin,
+                         ContentTypeMixin,
+                         viewsets.GenericViewSet):
+
+    queryset = Comment.objects.all().order_by('-created')
+    serializer_class = CommentModerateSerializer
+    # permission_classes = (ViewSetRulesPermission,)
```
Comment (phillimorland) [inline adhocracy4/comments/api.py]: do we not need this?

Comment (fuzzylogic2000) [reply adhocracy4/comments/api.py]: Yes! Or not... You found the bit where there is still work to do! I need to make it work with the rules to be more specific. Currently everyone is allowed to edit the is_moderator_marked via the api.

**Pull request merged**